src/action/interactive.rs

```diff
@@ -435,7 +437,6 @@ impl UserPicked {
                 }
             }
 
-            drop(guard);
```

Comment:
That `drop()` is unnecessary because `drop` for `ScopedRaw` in line 424 is called when it goes out of scope and now it's ensured that all earlier `guards` are properly dropped right? Just for my understanding :)

Reply:
I am not sure if the current approach is sane, there seem to be a few artificats in the cli, cursor showing and lines being off by one, just because we leave raw mode, so this might have to be reverted eventually.